src/renderers/dom/client/syntheticEvents/__tests__/SyntheticEvent-test.js

```diff
@@ -73,6 +73,7 @@ describe('SyntheticEvent', function() {
   });
 
   it('should be nullified if the synthetic event has called destructor', function() {
+    spyOn(console, 'error'); // accessing properties on destructored events logs warnings (tested elsewhere)
```

Comment:
We should still assert the warnings here.  The reason being that we want to know if we start emitting some unexpected warnings.  Right now, this test just swallows all warnings.